packages/react/src/auto/polaris/inputs/PolarisAutoRelationshipInput.tsx

```diff
@@ -0,0 +1,92 @@
+import { Combobox, Icon, Listbox, Tag, Text } from "@shopify/polaris";
+import { SearchIcon } from "@shopify/polaris-icons";
+import React from "react";
+import { Control } from "react-hook-form";
+import { ErrorWrapper } from "../../../utils.js";
+import { numberOfRecordsToLoad, useRelationshipInputController } from "../../hooks/useRelationshipInputController.js";
+
+export const PolarisAutoRelationshipInput = (props: {
```

Comment:
I get why this is one component we intend to use for all the relationships internally -- its the same component in multi mode or not multi mode. But, I think it'd be a clearer interface for users of these components to have one per relationship type instead of one for all relationships, since I think we generally represent the relationships as super different externally. Can we name the exports that they use and that we document things like `PolarisAutoBelongsToInput` and use this sucker under the hood?

Reply:
Updated such that there are individual fieldType based exports in the `index` file that point to the same `AutoRelationshipInput` component

```jsx
export const PolarisAutoBelongsToInput = (props: AutoRelationshipInputProps) => {
  return <PolarisAutoRelationshipInput {...props} />;
};
```